### PR TITLE
Jkl 136 msg server proofs test go

### PR DESCRIPTION
**Pull request by nuggetnchill**
`go test -v -run TestStorageTestSuite/TestPostProof`

Let me know if this needs more test cases or any adjustment.

### CODE REVIEW

File: x/storage/keeper/msg_server_proofs_test.go [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+	// Create provider account
+	testProvider, err := sdk.AccAddressFromBech32("cosmos1ytwr7x4av05ek0tf8z9s4zmvr6w569zsm27dpg")
+	suite.Require().NoError(err)
+
+	// Init Provider
+	_, err = msgSrvr.InitProvider(context, &types.MsgInitProvider{
+		Creator:    testProvider.String(),
+		Ip:         "198.0.0.1",
+		Totalspace: "1_000_000",
```
Comment (TheMarstonConnell) [inline x/storage/keeper/msg_server_proofs_test.go:125]: always forget you can do this, so clean

File: x/storage/keeper/msg_server_proofs_test.go [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+	// for tc 1 and 2
+	item, hashlist, err := CreateMerkleForProof(fileFromSP)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	// for tc 3: post proof from a different file
+	item2, hashlist2, err2 := CreateMerkleForProof(randomFile)
+	if err != nil {
+		fmt.Println(err2)
```
Comment (TheMarstonConnell) [inline x/storage/keeper/msg_server_proofs_test.go:188]: Can we replace all the `fmt.Println(err)` with a return statement to stop the test from passing?

Comment (TheMarstonConnell) [reply x/storage/keeper/msg_server_proofs_test.go:188]: Actually replace with `suite.Require().NoError(err)` instead of the entire if statement

**Review by TheMarstonConnell — CHANGES_REQUESTED** [review]

File: x/storage/keeper/msg_server_proofs_test.go (hunk 2 above)
Comment (nuggetnchill) [reply x/storage/keeper/msg_server_proofs_test.go]: got rid of all the fmt.Println(err), fmt.Printf() are cool tho?

**Review by TheMarstonConnell — APPROVED** [review]
we ball

**Pull request merged**